install_cardio.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+# encoding: utf-8
+# This script adds card.io to the swift example app.
+
+begin
+  gem "xcodeproj"
+rescue LoadError
+  system("gem install xcodeproj")
+  Gem.clear_paths
+end
+
+require 'xcodeproj'
+
+# puts '▸ Installing card.io – this may take a while'
+# open('Example/Cartfile', 'a') { |f|
+#   f.puts "github \"card-io/card.io-iOS-source\""
+# }
+# system('cd Example; carthage update --platform ios')
+
+project_path = 'Example/Stripe iOS Example (Simple).xcodeproj'
+project = Xcodeproj::Project.open(project_path)
+
+# add framework
+lib_path = 'Carthage/Build/iOS/CardIO.framework'
+libRef = project['Frameworks'].new_file(lib_path);
+framework_buildphase = project.objects.select {
+  |x| x.class == Xcodeproj::Project::Object::PBXFrameworksBuildPhase
+}[0];
+framework_buildphase.add_file_reference(libRef);
```

Comment:
Hmm, did some fiddling and reading, and I don't think this would work for our use case. As I understand it, weak linking is used for system frameworks – it conditionally links your app to the framework, so when your app runs on an OS version that doesn't have the framework, it won't crash. Even if a framework is weak linked, it still needs to be present on your machine in order to build the app.